PoGo.NecroBot.Logic/Tasks/CatchNearbyPokemonsTask.cs

```diff
@@ -17,6 +18,18 @@ public static class CatchNearbyPokemonsTask
     {
         public static void Execute(Context ctx, StateMachine machine)
         {
+            var invResp = ctx.Client.Inventory.GetInventory();
+            var invRespRes = invResp.Result;
+            if (invRespRes.Success)
+            {
+                var items = invResp.Result.InventoryDelta.InventoryItems;
+                if (items.Select(rawItem => rawItem.InventoryItemData.Item).Any(item => (item.ItemId == ItemId.ItemPokeBall || item.ItemId == ItemId.ItemGreatBall || item.ItemId == ItemId.ItemUltraBall || item.ItemId == ItemId.ItemMasterBall) && item.Count == 0))
```

Comment:
Doubt this is going to work, pretty much says if any of the pokeball types are 0 then we have no pokeballs